### PR TITLE
Noderunner fix RC

### LINKED ISSUE #8: Cookbook decompiler assumes double quotes
```
Line include_recipe 'java::set_attributes_from_version'
 does not contain an include_recipe statement
/Library/Ruby/Gems/2.0.0/gems/testbeat-0.5.4/lib/vagrant/cookbook_decompiler.rb:28:in `get_cookbook_name'
```

The regex needs to support both single and double quotes.

### PR DESCRIPTION
**Pull request by solsson**
Attempts to fix #8 and #9. Published as pr8 (inconsistent with this PR's number, but includes #7).

### CODE REVIEW

**Comment by takesson** [comment]
`SyntaxError: /var/lib/gems/2.2.0/gems/testbeat-0.6.0.pr8/lib/vagrant/noderunner.rb:239: syntax error, unexpected keyword_else, expecting keyword_end
/var/lib/gems/2.2.0/gems/testbeat-0.6.0.pr8/lib/vagrant/noderunner.rb:312: syntax error, unexpected end-of-input, expecting keyword_end
/var/lib/gems/2.2.0/gems/testbeat-0.6.0.pr8/lib/testbeat.rb:11:in `<top (required)>'
/var/lib/jenkins/jobs/cms-chef/workspace/Rakefile:8:in `<top (required)>'`

File: lib/vagrant/noderunner.rb [hunk 1]
```diff
@@ -218,8 +218,12 @@ def main(node: "labs01", provider: "virtualbox", retest: false, guestint: true,
...
       print "Recipes (rerun based on #{runlist_file}): "
       puts recipes
-      all_cookbooks = CookbookDecompiler.resolve_dependencies(recipes).to_a
-      puts "All cookbooks included: " + all_cookbooks.join(", ")
+      all_cookbooks = recipes
+      if dependencies
+        # Try to find dependencies for cookbooks identified at provision
+        all_cookbooks = CookbookDecompiler.resolve_dependencies(recipes).to_a
+        puts "All cookbooks included: " + all_cookbooks.join(", ")
+      else
```
Comment (takesson) [inline lib/vagrant/noderunner.rb]: @solsson end ?

**Comment by takesson** [comment]
@solsson Simple fix, I commented on the code. 

This release will address quite a few issues, good stuff!

**Comment by solsson** [comment]
Ok sorry, try `0.6.0.pr9`

**Comment by takesson** [comment]
Thanks! Tested, feel free to merge.

**Comment by solsson** [comment]
Released and published 0.6.0

**Pull request merged**